crates/sui-graphql-rpc/tests/snapshots/snapshot_tests__test__schema_sdl_export.snap

```diff
@@ -93,6 +93,18 @@ type CoinEdge {
 	cursor: String!
 }
 
+type Epoch {
+	epochId: Int!
+	systemStateVersion: BigInt!
+	protocolConfigs: ProtocolConfigs!
+	referenceGasPrice: BigInt!
+	systemParameters: SystemParameters!
+	stakeSubsidy: StakeSubsidy!
+	validatorSet: ValidatorSet!
+	storageFund: StorageFund!
+	safeMode: SafeMode!
```

Comment:
Many types added in this struct and others should actually be optional. See the corresponding draft schema entries [here](https://github.com/MystenLabs/sui/blob/main/crates/sui-graphql-rpc/schema/draft_schema.graphql#L395). Not the lack of exclamation mark on the type in the draft schema.
Fixing these should be as easy as changing `field_name: Type` to `field_name: Option<Type>` in the struct definitions, and wrapping the value assignments in `Some(...)`
```
type Epoch {
  epochId: Int!
  systemStateVersion: Int
  protocolConfigs: ProtocolConfigs
  referenceGasPrice: BigInt

  systemParameters: SystemParameters
  stakeSubsidy: StakeSubsidy
  validatorSet: ValidatorSet
  storageFund: StorageFund
  safeMode: SafeMode
```

Reply:
Will fix, thanks!